cni/network/network_windows.go

```diff
@@ -264,59 +263,55 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig, isIPv6Enabled bool) []p
 			protocol = policy.ProtocolUdp
 		}
 
+		hostIP, err := netip.ParseAddr(mapping.HostIp)
```

Comment:
is hostIP non-empty for all portmapping enabled pods?  If hostIP is empty, Is that valid case?

Reply:
it's slightly possible that if we have portmapping but hostIP is empty, for example:
```
   ports:
   - hostPort: 54321
     containerPort: 8080
     protocol: TCP
     hostIP: ""
 ```
 But it's mostly not a case, what do you think?

Reply:
Yes it's not a required field

Reply:
@paulyufan2 are we able to test the scenario that host IP in our pipeline?